### ISSUE TITLE
branding

### ISSUE THREAD

**Issue by lytico** [dotdevelop/dotdevelop#21]
this is about branding
about branches see: #41

**Comment by hardhub** [comment]
Will it be rebranded?

monodevelop.com => dotdevelop.com

as well as product name?

**Comment by hardhub** [comment]
or maybe dotdevelop.org is better for that

**Comment by lytico** [comment]
dotdevelop.net ?

**Comment by hardhub** [comment]
Yes, also makes sense. For me all above is OK. But let's check what community thinks :)

**Comment by lytico** [comment]
and ms. maybe, if we have a working fork of monodevelop, it could be a chance to be integrated in mono/monodevelop again. but that's a wish...

**Comment by scherenhaenden** [comment]
Hi guys,

i got a new job and it is in Java in a huge company and it is consuming my whole time + Family and all the stuff. 

Anyways, I'm very well surprised and happy about how is it looking now. 

I would vote vor dotdevelop.net/org 
Maybe you guys need some money support? 

Keep rocking, ill try to be following the project!

**Comment by lytico** [comment]
it's a pity, but
good luck in your new job!

concerning money: in a longterm, i think we'll need fundraising. the project is quite big, and if we want professional support, it's not to make as a "hobby".

**Comment by knuxbbs** [comment]
> 
> 
> and ms. maybe, if we have a working fork of monodevelop, it could be a chance to be integrated in mono/monodevelop again. but that's a wish...

I have the same thought. MonoDevelop must survive, despite Microsoft's interests.

**Comment by knocte** [comment]
I don't think keeping the name MonoDevelop is a good idea, because the Mono brand is fading (given that .NET6 will include Mono under the hood, and essentially make it be a fusion of the two projects ).


>if we have a working release running on .net 5, it should become the main branch:
>dotdevelop

Why not master?


>  i think we'll need fundraising.

I think I know someone that might be interested to fund you (or other dev) part-time work on this. Send me an email.

**Comment by lextm** [comment]
@knocte Sticking to the Mono/MonoDevelop brand isn't a bad thing even if Microsoft is losing interest in that.

For decades, the Mono brand has been the flagship for open source .NET, and spread everywhere.

With a working build from this repo on major Linux distributions (or at least Ubuntu), I think this group of community members has a chance to take over MonoDevelop assets directly from Microsoft via proper negotiation.

**Comment by knocte** [comment]
@lextm those are very good points, however, I think having Mono in the name would bring some deal of confusion (making them think it's not .NETCore/.NET5 compatible), which IMO makes the option not worth it.

**Comment by knuxbbs** [comment]
Monogame runs on .NET Core and I don't think that this was a problem to the users.

**Comment by knocte** [comment]
was? I'm mainly talking about a transition that hasn't happened yet (.NET6 including the Mono runtime, so Mono brand/product fading since that moment). So the effects of that possible confusion could only be seen after about a year.

**Comment by knocte** [comment]
The name "DotDevelop" is a good mix between two concepts: "DotNet" and "MonoDevelop". However, it doesn't really evoke as the name of an IDE, or the name of a program at all. How about we come up with new names? My first proposals would be: "DotStudio" (DotNet+VisualStudio) or "NetBrains" (resembling NetBeans/JetBrains lol).
